go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/infostellarinc/stellarcli
 require (
 	cloud.google.com/go v0.34.0 // indirect
 	github.com/MakeNowJust/heredoc v0.0.0-20171113091838-e9091a26100e
+	github.com/cenkalti/backoff v2.1.1+incompatible
```

Comment:
Not a hard reason for picking a library but we'd usually prefer something on awesome-go to something that isn't. Otherwise we should check the code and make sure it is a well written library.

Here's one that is on awesome-go (so presumably has enough vetting that we don't have to ourselves)

https://github.com/ssgreg/repeat

Reply:
I haven't checked awesome-go, and it is good to know the site to pick up libraries with quality. For this particular case, I saw couple of libraries and chose the one which have most starts. Comparing to ssgreg/repeat which has 53 stars 1 contributer, 1262 starts + 15 contributers seems persuasive to choose together with another reason where back off based on the logic of Google's HTTP Client Library.